releasenotes/notes/add-new-symbolic-pulses-4dc46ecaaa1ba928.yaml

```diff
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    The ``SymbolicPulse`` library was extended. The new pulses in the library are:
+
+      * :class:``~qiskit.pulse.library.Sin``
+      * :class:``~qiskit.pulse.library.Cos``
+      * :class:``~qiskit.pulse.library.Sawtooth``
+      * :class:``~qiskit.pulse.library.Triangle``
+
+    The new functions return a ``ScalableSymbolicPulse``. With the exception of the ``Sawtooth`` phase,
```

Comment:
It's better to describe how it is different.

Reply:
Thank you.